### ISSUE TITLE
Benchmarking on Meter-ML

### ISSUE THREAD

**Issue by bengmstrong** [IBM/terramind#14]
I'm trying to use terramind to produce embeddings like in the benchmarking results shown in the TerraMind paper (Table 5). In the paper it mentions you average over the patch embeddings. Can you elaborate on the process used to benchmark on the S2-L1C imagery? So far I've done this:

```
model = BACKBONE_REGISTRY.build(
                    "terramind_v1_base",
                    modalities=["S2L1C"],
                    pretrained=True,
                )
  model.head = DoubleMeanPoolingHead()
```

where DoubleMeanPoolingHead() takes the raw output of the backbone (a list of tensors of size [1, 196, 768]) and averages over the tensors twice to produce a single tensor of size [1, 768] which I use as the embedding. I also normalize the S2L1C imagery using the parameters in `terratorch.models.backbones.terramind.model.terramind_register`.

Thanks for your help! I'm excited to use terramind to test out some emebddings!

![Image](https://github.com/user-attachments/assets/d8055e95-f6c6-4302-be41-eb08900011cc)

**Comment by blumenstiel** [comment]
Hi @bengmstrong, we just averaged the tensor over the second dimension. If your head is doing the same, then the results should be compareable. We also used the pre-training statisitics for normalisation.
For the experiments, we sampled one example per class from the train split and evaluated the classification over the full test split. In the appendix we provide more results using all classes and also sampling 5 examples per class (5-shot).
